src/actions/sources/newSources.js

```diff
@@ -40,11 +40,14 @@ function createOriginalSource(
 }
 
 // TODO: It would be nice to make getOriginalURLs a safer api
-async function loadOriginalSourceUrls(sourceMaps, generatedSource) {
+async function loadOriginalSourceUrls(dispatch, sourceMaps, generatedSource) {
   try {
```

Comment:
this function might have outlived it's usefulness. Perhaps we kill it and inline it back into `loadSourceMap`

Reply:
I think that would make sense.